samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```diff
@@ -35,9 +35,9 @@
     <MauiImage Include="Resources\Images\*" />
     <EmbeddedResource Include="Resources\Embedded\*" />
 
-    <PackageReference Include="CommunityToolkit.Maui.Markup" Version="1.0.0-rc1" />
+    <PackageReference Include="CommunityToolkit.Maui.Markup" Version="1.0.0-rc2" />
     <PackageReference Include="CommunityToolkit.Mvvm" Version="8.0.0-preview3" />
-    <PackageReference Include="Microsoft.Extensions.Http" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="6.0.2-mauipre.1.22102.15" />
```

Comment:
it has the maui in the version name. I think it suits better

Reply:
Makes sense. Now that MAUI targets .NET SDK v6.0.300, I assume we'll be able to update this to Microsoft.Extensions.Http v6.0.3 once it's released